### PR TITLE
[WIP] Updating the simple_form dependency to use 5.0.x releases and releases version 5.0.1

### PR DESCRIPTION
**Pull request by jrgriffiniii**
This is issued in response to https://github.com/plataformatec/simple_form/security/advisories/GHSA-r74q-gxcg-73hx

### CODE REVIEW

File: hydra-editor.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.add_dependency "almond-rails", '~> 0.1'
   s.add_dependency "cancancan", "~> 1.8"
   s.add_dependency "rails", ">= 5", "< 6"
-  s.add_dependency "simple_form", '~> 4.1.0'
+  s.add_dependency "simple_form", '>= 5.0'
```
Comment (no-reply) [inline hydra-editor.gemspec]: I think dropping support for `4.1.0` would require a major version. `s.add_dependency "simple_form", '>= 4.1.0', '< 6.0'` instead?

**Comment by jrgriffiniii** [comment]
https://github.com/ma2gedev/faraday-encoding/commit/dc749230d4dd71dedc4b2703a7bb62ac1ea8f7c9 seems to be blocking the tests from passing given that `active-fedora` is locked to release 0.0.4: https://github.com/samvera/active_fedora/blob/master/active-fedora.gemspec#L25

I can offer a pull request loosening the dependency, and then proceed.  I apologize for this.

**Comment by jrgriffiniii** [comment]
https://github.com/samvera/active_fedora/compare/master...jrgriffiniii:13.1.1 will need to be merged and 13.1.1 released, and then this will no longer be a WIP.

**Comment by jrgriffiniii** [comment]
Diagnosing the failing builds requires that someone other than myself be in the position to restart CircleCI.  I'm closing this and issuing a separate PR from a `samvera` branch.

**Pull request closed without merging**